autogpt/config/config.py

```diff
@@ -41,7 +41,24 @@ def __init__(self) -> None:
         self.fast_llm_model = os.getenv("FAST_LLM_MODEL", "gpt-3.5-turbo")
         self.smart_llm_model = os.getenv("SMART_LLM_MODEL", "gpt-4")
         self.fast_token_limit = int(os.getenv("FAST_TOKEN_LIMIT", 4000))
-        self.smart_token_limit = int(os.getenv("SMART_TOKEN_LIMIT", 8000))
+
+        # It's better to set max limits and be explicit for better user experience IMO
+        model_limits = {
+            "gpt-4-32k": 32000,
+            "gpt-4": 8000,
+            "gpt-3.5-turbo": 4000,
+            "gpt-3.5-turbo-0301": 4000,
+        }
```

Comment:
Look through the files, I think these are already stored somewhere

Reply:
https://github.com/Significant-Gravitas/Auto-GPT/pull/4128/commits/b3ba8581c157a23f1c0228f619945521dc38e116

kk I pull in from the constants in another file now

Reply:
[2ad9647](https://github.com/Significant-Gravitas/Auto-GPT/pull/4128/commits/2ad96477c273db109eba59397873f47407f722f2)